lld/COFF/Chunks.h

```diff
@@ -624,7 +624,8 @@ class ImportThunkChunkARM64EC : public ImportThunkChunk {
   MachineTypes getMachine() const override { return ARM64EC; }
   void writeTo(uint8_t *buf) const override;
 
-  Defined *exitThunk;
+  Defined *exitThunk = nullptr;
```

Comment:
I guess the null initialization of `exitThunk` is an unrelated change? Although probably good in itself.

Reply:
Yes, I will split it.